.appveyor/config.yml

```diff
@@ -4,7 +4,7 @@ environment:
   ANDROID_BUILD_VERSION: 28
   ANDROID_TOOLS_VERSION: 28.0.3
 
-  GRADLE_OPTS: -Dorg.gradle.daemon=false
```

Comment:
I don't see this necessary, can you explain why ?

Reply:
Hi @gengjiawen - Thanks for taking time to review the PR!
Manually increasing the heap space was necessary because of the way the build is run on AppVeyor. There is no `gradle.properties` file in the root directory of the repo (even though it contains the Gradle wrapper scripts and Gradle build files). You can see it _directly_ runs the `assembleRelease` task in `RNTester/android` - It might not immediately be obvious, but when called that way it will *not* pick up the Gradle settings from that directory, and instead use the [default heap space of only 64MB](https://github.com/facebook/react-native/blob/master/gradlew.bat#L33)! Previously it failed because it was running out of heap space. However, it looks like the AppVeyor CI build was disabled in 382cb05ae6ccf96355d297bcf54055305c2a7bac, so this line modification might not be needed anymore (at least for now). Let me try to remove it.

Reply:
It's removed because android failed to run due to low memory. We intend to re-enable it later.
So maybe just leave it be.